expyfun/_utils.py

```diff
@@ -426,12 +418,21 @@ def verbose_dec(function, *args, **kwargs):
         return ret
 
 
+def _has_avbin():
+    try:
+        from pyglet.media.avbin import AVbinSource
+        del AVbinSource
+        return True
+    except ImportError:
+        return False
+
+
 _is_appveyor = (os.getenv('APPVEYOR', 'False').lower() == 'true')
 requires_pandas = skipif(has_pandas is False, 'Requires pandas')
 requires_h5py = skipif(has_h5py is False, 'Requires h5py')
 requires_joblib = skipif(has_joblib is False, 'Requires joblib')
 requires_opengl21 = skipif(_is_appveyor, 'Appveyor OpenGL too old')
-requires_avbin = skipif(has_avbin is False, 'Requires AVbin')
+requires_avbin = skipif(_has_avbin() is False, 'Requires AVbin')
```

Comment:
Doesn't make a difference when dealing with bools here.

I started doing `not X` instead of `X is False` to allow for e.g. `0` vs `1` as parameters to behave more naturaly, because these evaluate in the expected way with `not X` (0->True, 1->False) whereas `X is False` will not (0->False, 1->False). The only time I do `X is True` or `X is False` now is when I actually want to differentiate e.g. `None`, `False`, `0`, `[]`, `''`, etc., which all pass the same way with `if not X` but will be different for `if X is False`.